### PR TITLE
Added contribution.md and updated readme.md

### PR DESCRIPTION
**Pull request by vismitap**
Hello! I have added contribution.md in the root file and have also done necessary changes in readme.md. Please review this PR and suggest improvements if any. It was amazing to contribute to this repo!!

### CODE REVIEW

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+We welcome all contributions to the Hydrator project. The contribution can be a [issue report](https://github.com/devcer/Hydrator/issues) 
+or a [pull request](https://github.com/devcer/Hydrator/pulls).
+
+## Contribution workflow:
+
+
+1. Check that there isn't [already an issue](https://github.com/devcer/Hydrator/issues) about your idea to avoid duplicating work.
+    * If there isn't one already, please create one so that others know you're working on this
+
```
Comment (devcer) [inline CONTRIBUTION.md:11]: @vismitap Can you also add star the repo to the workflow? May be an additional step below step 1?

**Comment by devcer** [comment]
@vismitap Added a comment. rest all looks good. Please sign your commits so that all checks are passed before I manually review.

File: CONTRIBUTION.md (hunk 1 above)
Comment (vismitap) [reply CONTRIBUTION.md:11]: Sure! Please give me some time, working on that!

Comment (vismitap) [reply CONTRIBUTION.md:11]: @devcer I searched about this and could not find any method to link the star the repo thing to the readme file.  The maximum we can do is link the /stargazers to view the people who have starred the repo or link it to the original page (https://github.com/devcer/Hydrator). There's no direct way to star the repo from the readme itself, the user will have to go to the original repo page and star it from there. Should I go ahead with adding the original repo 
link?

Comment (devcer) [reply CONTRIBUTION.md:11]: @vismitap you can do exactly what you did with fork the repo step. Maybe I wasn't clear before, sorry bout it.
`Star the [devcer/Hydrator repository](https://github.com/devcer/Hydrator) to your GitHub account.`
Add emoticons for star, fork, and clone if possible.

Comment (vismitap) [reply CONTRIBUTION.md:11]: Oh okay!

**Review by devcer — APPROVED** [review]

**Comment by devcer** [comment]
@vismitap Please sign your commits. I'm unable to merge them without signed commits.

**Comment by vismitap** [comment]
I was new to the signing of commits. Thank you for introducing this to me. Just created a GPG key, working on that...almost done

**Comment by devcer** [comment]
@vismitap If you are using terminal for commits then you can sign them with `-S` flag. Read here -> https://docs.github.com/articles/about-gpg/

**Comment by vismitap** [comment]
Yeah had read about that command but that is used for new commits. I wanted to sign the previous ones. I tried all possible things like filtering branches, amending, creating a new branch, and copying, etc and messed up badly. Didn't want to push commit in this PR as it would have been very difficult for you to merge. Had an amazing learning experience though. I have raised a new PR with signed commit and prev changes. Please review that PR and close this one. Sorry for the inconvenience.

**Pull request closed without merging**